### PR TITLE
Student table edit students

### PR DESCRIPTION
**Pull request by TheTallJerry**
## Motivation and Context

We should make the student table's dropdown and student table's edit option's functionalities consistent. 

## Your Changes

**Description**:
- Allows instructors to bulk update students' sections in student table
#### Current student table dropdowns
old
#### Updated student table dropdowns
new

- Allows instructors to update individual student's status (inactive / active) from the edit page, accessible via the "edit" option in student table
#### Current student edit page
old student edit
#### Updated student edit page
new student edit

**Type of change** (select all that apply):

- [x] New feature (non-breaking change which adds functionality)
- [x] Test update (change that modifies or updates tests only)

## Testing

Ran rspec and Jest tests. 

## Checklist

- [x] I have performed a self-review of my own code.
- [x] I have verified that the pre-commit.ci checks have passed. 
- [x] I have verified that the CI tests have passed. 
- [x] I have reviewed the test coverage changes reported on Coveralls. 
- [x] I have added tests for my changes. 
- [x] I have updated the Changelog.md file. 
- [x] I have made changes to the documentation and linked to that pull request below. 

### Pull request to make documentation changes (if applicable)
https://github.com/MarkUsProject/Wiki/pull/142

### CODE REVIEW

File: app/assets/javascripts/Components/student_table.jsx [hunk 1]
```diff
@@ -241,6 +241,9 @@ class StudentsActionBox extends React.Component {
             onChange={this.inputChanged}
             data-testid={"student_action_box_update_section"}
           >
+            <option key={"none"} value={""}>
+              {I18n.t("students.instructor_actions.no_section")}
```
Comment (david-yz-liu) [inline app/assets/javascripts/Components/student_table.jsx:245]: This string is good, and I think we should use it in the student form as well (rather than using an empty string)

File: app/views/students/_student_form.html.erb [hunk 2]
```diff
@@ -21,6 +21,11 @@
                   remote: true,
                   class: 'button inline-button' %>
     </span>
+    <%= f.label :hidden, t('students.update_status') %>
```
Comment (david-yz-liu) [inline app/views/students/_student_form.html.erb:24]: To match the other labels on the form this string should just be "Status"

File: app/views/students/_student_form.html.erb [hunk 3]
```diff
@@ -21,6 +21,11 @@
...
+    <span id='set-active'>
```
Comment (david-yz-liu) [inline app/views/students/_student_form.html.erb:25]: I'm not sure why you included a `span` here, the layout should be okay with just the `select` tag (I think)?

**Review by david-yz-liu — COMMENTED** [review]
@TheTallJerry nice job.

1. You should update the Changelog and the wiki.
2. You should add tests for the new features on the students table (setting section to nil) and student form (updating `hidden` status)

File: app/views/students/_student_form.html.erb (hunk 3 above)
Comment (TheTallJerry) [reply app/views/students/_student_form.html.erb:25]: We have styling in place that will fill the select with more space without the span tag. Removing the tag in this case would look like:
Capture
I feel it's easier to just have the tag here.

File: spec/controllers/students_controller_spec.rb [hunk 4]
```diff
@@ -41,6 +59,61 @@
       end
     end
 
+    describe '#bulk_modify' do
```
Comment (TheTallJerry) [inline spec/controllers/students_controller_spec.rb:62]: I realized bulk_modify is actually entirely untested, so in addition to tests for bulk setting sections, I added tests for bulk setting everything else too.

**Review by david-yz-liu — APPROVED** [review]
@TheTallJerry thanks!
